server/service/src/sync/test/integration/remote/clinician.rs

```diff
@@ -111,21 +107,6 @@ impl SyncRecordTester for ClinicianRecordTester {
                 PullUpsertRecord::Clinician(row.clone()),
             ]),
         });
-        // STEP 3 - delete
```

Comment:
Not sure if we should be deleting clinicians?

Reply:
I don't think we should be